### DISCUSSION TITLE
Grammar cases in Polish

### DISCUSSION THREAD

**Discussion by andr1972** [explosion/spaCy#12389] [Help: Other Questions] [answered]
Is possible detect grammars cases of nouns in sentence in Polish any way?

**Comment by thomashacker** [comment, marked as answer]
Hey, thanks for your question.
Grammar cases are included in the [morphological features](https://spacy.io/usage/linguistic-features#morphology) provided by the [morphologizer](https://spacy.io/api/morphologizer) component. But in general, their accuracy isn't very high.

**Comment by andr1972** [comment]
Thanks, 
I am new in spaCy - what is way to install components like morphologizer and if is list of available spaCy components?

**Reply by thomashacker** [reply]
The pretrained [polish models](https://spacy.io/models/pl) include the `morphologizer` component. You can read more about how to access its generated information [here in our docs](https://spacy.io/usage/linguistic-features#morphology).
